doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -1178,8 +1196,11 @@ Confidentiality and Integrity (source `RM 7.9.3 <../../../ref_model/chapters/cha
 Table 2-8: Reference Model Requirements: Confidentiality and Integrity
 Requirements
 
-Workload Security (source `RM 7.9.4 <../../../ref_model/chapters/chapter07.md#794-workload-security>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Workload Security
+^^^^^^^^^^^^^^^^^
+
+(source `RM 7.9.4
+<../../../ref_model/chapters/chapter07.md#794-workload-security>`__)
```

Comment:
@collivier error, It should be 
:ref:`ref_model/chapters/chapter07:workload security`